HybrasylXML/XSD/Castables.xsd

```diff
@@ -162,7 +163,7 @@
   <xs:complexType name="Intent">
     <xs:attribute name="UseType" type="actions:SpellUseType" default="NoTarget" use="optional"/>
     <xs:attribute name="Radius" type="xs:unsignedByte" default="0" />
-    <xs:attribute name="Direction" type="actions:IntentDirection" default="Front" />
+    <xs:attribute name="Direction" type="actions:IntentDirection" default="None" use="optional"/>
```

Comment:
How is optional use a thing for this?

Reply:
`optional` here has no real effect in xsd2code, so I suppose we could drop it. Now that there is `None` it is less of a concern.